src/Validation/composer.json

```diff
@@ -1,21 +1,35 @@
 {
     "name": "cakephp/validation",
     "description": "CakePHP Validation library",
+    "type": "library",
+    "keywords": [
+        "cakephp",
+        "validation",
+        "data validation"
```

Comment:
It's kind of SEO, I guess.

This way, it would be shown on this list, too:
https://packagist.org/search/?tags=data%20validation

Otherwise, not.